docs/pages/packages/node-utils.mdx

```diff
@@ -0,0 +1,122 @@
+import { Tabs, Tab } from '../../components/tabs'
+
+# Edge Runtime Node Utils **Alpha version**
+
+The **@edge-runtime/node-utils** package contains utilities to run web compliant code into a Node.js environment.
+
+**Please note this is an alpha version.**
+
+## Installation
+
+<Tabs items={['npm', 'yarn', 'pnpm']} storageKey='selected-pkg-manager'>
+  <Tab>```sh npm install @edge-runtime/node-utils --save ```</Tab>
+  <Tab>```sh yarn add @edge-runtime/node-utils ```</Tab>
+  <Tab>```sh pnpm install @edge-runtime/node-utils --save ```</Tab>
+</Tabs>
+
+This package includes built-in TypeScript support.
+
+## Usage
+
+```js
+import { createServer } from 'node:http'
+import { buildToNodeHandler } from '@edge-runtime/node-utils'
+
+// 1. builds a transformer, using Node.js@18 globals, and a base url for URL constructor.
+const transformToNode = buildToNodeHandler(global, {
+  origin: 'http://example.com',
+})
+
+const server = await createServer(
+  // 2. takes an web compliant request handler, that uses Web globals like Request and Response,
+  // and turn it into a Node.js compliant request handler.
+  transformToNode((req) => new Response(req.body))
+)
+
+// 3. start the node.js server
+await server.listen()
```

Comment:
server.listen isn't a promise; for doing that, you need to wait for the `listening` event:

```
server.listen()
await once(server, 'listening')
const { address, port } = server.address()
```